lib/trento.pm

```diff
@@ -149,8 +264,8 @@ sub az_vm_ssh_cmd {
     my ($self, $cmd_arg, $vm_ip_arg) = @_;
     record_info('REMOTE', "cmd:$cmd_arg");
 
-    # undef comparision operator
-    my $vm_ip = $vm_ip_arg // az_get_vm_ip();
+    # Undef comparison operator
+    my $vm_ip = $vm_ip_arg // get_trento_ip();
```

Comment:
You could also do:
```
$vm_ip_arg //= get_trento_ip();
```
And that way you avoid having 2 variables (`$vm_ip` and `$vm_ip_arg`) which basically have the same info.